src/main/java/az/edu/turing/domain/dao/impl/database/FlightDaoPostgres.java

```diff
@@ -0,0 +1,95 @@
+package az.edu.turing.domain.dao.impl.database;
+
+import az.edu.turing.config.ConnectionHelper;
+import az.edu.turing.domain.dao.inter.FlightDao;
+import az.edu.turing.domain.entity.FlightEntity;
+
+import java.sql.CallableStatement;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+public class FlightDaoPostgres extends FlightDao {
+
+    ConnectionHelper connectionHelper;
+
+    public FlightDaoPostgres(ConnectionHelper connectionHelper) {
+        this.connectionHelper = connectionHelper;
+        String createTableQuery = "CREATE TABLE IF NOT EXISTS flights (" +
+                "id UUID PRIMARY KEY NOT NULL," +
+                "departure_point VARCHAR(255) NOT NULL," +
```

Comment:
I think it would be better if we made it unique. (column  departure_point and destination_point)

Reply:
why? it is possible to exist two or more flights from Kiev to Baku. But on different times.

Reply:
good idea! then we can make the departure_time column unique